lms/templates/problem.html

```diff
@@ -5,38 +5,106 @@
 %>
 
 <%namespace name='static' file='static_content.html'/>
-<h3 class="hd hd-2 problem-header">
+<h3 class="hd hd-2 problem-header" id="${ id }-problem-title" aria-describedby="${ id }-problem-progress" tabindex="-1">
```

Comment:
We need the id of this h3 element to refer it from the [section element ](https://github.com/edx/edx-platform/pull/13550/files#diff-38722d7e328c77e747d36e85e7ab2c0fR265) for a11y purpose. We already have `short_id` at some point and then just need to pass it to the response type but for `id` we have to pass it from step 1 i.e. passing it to LonCAPAProblem and then passing it to the extract_html and then to response type but i would prefer to use short_id since we're already using it for response types and LonCAPAProblem https://github.com/edx/edx-platform/blob/master/common/lib/capa/capa/capa_problem.py#L154. You can back track this id in `capa_base.py:314`.